src/util/fastClick.ts

```diff
@@ -27,7 +28,7 @@ const fastClick = isTouch
           const y = e.touches[0].clientY
           touchStart = { x, y }
         }
-
+        Haptics.impact({ style: ImpactStyle.Light })
```

Comment:
It might make more sense to place the hapic in `onTouchEnd`, since that is where the handler fires. Then the haptic feedback will occur at the same time as the execution of the handler. If done in `onTouchStart`, there will be a gap between the haptic and the execution, and haptics will unintentionally occur on cancelled clicks (such as a drag or scroll).

Reply:
Great idea. I immediately saw the touch triggers happening at regular intervals after this change.